sunpy/database/tables.py

```diff
@@ -374,42 +377,31 @@ def _from_fido_search_result_block(cls, sr_block, default_waveunit=None):
         """
         # All attributes of DatabaseEntry that are not in QueryResponseBlock
         # are set as None for now.
-        source = sr_block.get('Source', None)
-        provider = sr_block.get('Provider', None)
-        physobs = sr_block.get('Physobs', None)
+        source = sr_block.get('Source')
```

Comment:
silly question - does this now mean there will always be these attributed in the table? what is the change here for?

Reply:
The None was redundant, the default is None anyway.